ide-support/revliburl.livecodescript

```diff
@@ -73,6 +80,49 @@ on libraryStack
    initialiseCustomProps
 end libraryStack
 ------------------------------------------------
+-- CW-2016-06-11: [[ External driver support ]] Add function to select external driver for use with libUrl.
+## Set driver to use for libUrl
+on libUrlSetDriver pDriver
```

Comment:
Hmm... I was thinking that we could get away with start using in IDE and including in standalone (not sure about the start using mechanism for included stacks there but I'll look into it) if there's a stackfile with `lib` in the filename in the external directory. In this case the library can set itself as the driver in libraryStack and clear itself in releaseStack. At some point we need some kind of manifest to declare what to do with ambiguous files but for the moment between mergExt and tsNet we only have `demo.livecode` and `tsnetliburl.livecodescript` files to deal with. Given we know libURL drivers are a thing now we could if we wanted declare that any file named `*.liburldriver.livecode*` be a driver and initialised on libURL but I'm not sure that's necessary.